loky/backend/context.py

```diff
@@ -8,93 +8,49 @@
+
 _DEFAULT_START_METHOD = None
 
 # Cache for the number of physical cores to avoid repeating subprocess calls.
 # It should not change during the lifetime of the program.
 physical_cores_cache = None
 
-if sys.version_info[:2] >= (3, 4):
-    from multiprocessing import get_context as mp_get_context
-    from multiprocessing.context import assert_spawning, set_spawning_popen
-    from multiprocessing.context import get_spawning_popen, BaseContext
-
-    START_METHODS += ['spawn']
-    if sys.platform != 'win32':
-        START_METHODS += ['fork', 'forkserver']
-
-    def get_context(method=None):
-        # Try to overload the default context
-        method = method or _DEFAULT_START_METHOD or "loky"
-        if method == "fork":
-            # If 'fork' is explicitly requested, warn user about potential
-            # issues.
-            warnings.warn("`fork` start method should not be used with "
-                          "`loky` as it does not respect POSIX. Try using "
-                          "`spawn` or `loky` instead.", UserWarning)
-        try:
-            context = mp_get_context(method)
-        except ValueError:
-            raise ValueError(
-                f"Unknown context '{method}'. Value should be in "
-                f"{START_METHODS}."
-            )
-
-        return context
-
-else:
-    if sys.platform != 'win32':
-        import threading
-        # Mechanism to check that the current thread is spawning a process
-        _tls = threading.local()
-        popen_attr = 'spawning_popen'
-    else:
-        from multiprocessing.forking import Popen
-        _tls = Popen._tls
-        popen_attr = 'process_handle'
-
-    BaseContext = object
-
-    def get_spawning_popen():
-        return getattr(_tls, popen_attr, None)
-
-    def set_spawning_popen(popen):
-        setattr(_tls, popen_attr, popen)
 
-    def assert_spawning(obj):
-        if get_spawning_popen() is None:
-            raise RuntimeError(
-                f'{type(obj).__name__} objects should only be shared between '
-                'processes through inheritance'
-            )
-
-    def get_context(method=None):
-        method = method or _DEFAULT_START_METHOD or 'loky'
-        if method == "loky":
-            return LokyContext()
-        elif method == "loky_init_main":
-            return LokyInitMainContext()
-        else:
-            raise ValueError(
-                f"Unknown context '{method}'. Value should be in "
-                f"{START_METHODS}."
-            )
+def get_context(method=None):
+    # Try to overload the default context
+    method = method or _DEFAULT_START_METHOD or "loky"
+    if method == "fork":
+        # If 'fork' is explicitly requested, warn user about potential
+        # issues.
+        warnings.warn("`fork` start method should not be used with "
+                        "`loky` as it does not respect POSIX. Try using "
+                        "`spawn` or `loky` instead.", UserWarning)
```

Comment:
At some point we will use black but in the mean time, this is better :)

```suggestion
        warnings.warn("`fork` start method should not be used with "
                      "`loky` as it does not respect POSIX. Try using "
                      "`spawn` or `loky` instead.", UserWarning)
```